From c79657496326c10f81cf80a8c9a0e55bc2ba2798 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Mon, 7 Dec 2020 14:34:09 +0100
Subject: [PATCH] Remove -Werror

It's doing more harm than good.

It makes it a pain to update anything (compiler, dependencies etc) that
may introduce new warnings. It also makes it hard to use a compiler that
produces a lot of warnings, e.g. clazy
---
 src/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 301c05e..99b0a87 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -33,12 +33,6 @@ add_library(
     ${sources} ${pluginData}
 )
 
-target_compile_options(
-    kquickimageeditorplugin
-    PRIVATE
-        -Werror
-)
-
 target_link_libraries(
     kquickimageeditorplugin
     PRIVATE
-- 
2.30.0

